DEPS

```diff
@@ -473,6 +468,28 @@ deps = {
      'dep_type': 'cipd',
    },
 
+  'src/buildtools/{host_os}-x64': {
+    'packages': [
+      {
+        'package': 'gn/gn/${{platform}}',
+        'version': 'git_revision:bdb0fd02324b120cacde634a9235405061c8ea06'
+      },
+    ],
+    'condition': 'host_os == "mac" or host_os == "linux"',
```

Comment:
do we need to pull down `gn` for Windows?

Reply:
Yes. At [`buildtools/win`](https://github.com/flutter/engine/blob/98241b7b05aa605755dc87ce33c7f6fb189d4863/tools/gn#L358). You could probably simplify this and get rid of the `x64` suffix for all platforms and modify the `tools/gn` script.

Reply:
I'd rather do that as a separate or then.  We would have to update buildroot to know where the compilers are.

Probably we can just drop the is specific name completely for the local copy

Reply:
I've changed this around to move gn to where depot_tools will know how to look for it without requiring an OS specific path.  It feels a little strange to put it there - if we think it doesn't work for us we can move it somewhere else and just not use depot_tools' gn, but I like being able to use depot_tools for this.